blueocean-dashboard/src/main/js/components/karaoke/components/InputStep.jsx

```diff
@@ -149,6 +162,9 @@ const { object, shape } = PropTypes;
 InputStep.propTypes = {
     step: shape().isRequired,
     classicInputUrl: object,
+    pipeline: PropTypes.object,
```

Comment:
I feel you should create the `classicInputUrl` in the calling component and you would not have to change this. If you prefer the current way then you should drop `classicInputUrl`  and the `this.props.classicInputUrl`  usage (only used now in comments)